CONTRIBUTING.rst

```diff
@@ -99,19 +99,19 @@ to ask clarifying questions on the ticket as needed, too, if anything is unclear
 Step 1: Sign a Contribution Agreement
 =====================================
 
-Before edX can accept any code contributions from you, you'll need to sign
-the `individual contributor agreement`_ and send it in. This confirms
-that you have the authority to contribute the code in the pull request and
-ensures that edX can relicense it.
+Before edX can accept any code contributions from you, you'll need to sign the
+`individual contributor agreement`_. This confirms that you have the authority
+to contribute the code in the pull request and ensures that edX can re-license
+it.
+
```

Comment:
I think we should mention entity agreement, something along the lines of "If you contribute code on behalf of your employer or another institution you are affiliated with, please reach out by email to legal@edx.org to request the entity contributor agreement. Then, once your entity has signed the agreement, we will be able to review and merge your code."

Reply:
done